### PR TITLE
add and implement interface of TableColumns

### PR DESCRIPTION
**Pull request by iamlinjunhong**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5590

## What this PR does / why we need it:

### CODE REVIEW

**Review by XuPeng-SH — APPROVED** [review]
lgtm

**Review by daviszhen — APPROVED** [review]

File: pkg/vm/engine/memoryengine/table.go [hunk 1]
```diff
@@ -172,6 +172,27 @@ func (t *Table) GetPrimaryKeys(ctx context.Context) ([]*engine.Attribute, error)
...
 }
 
+func (t *Table) TableColumns(ctx context.Context) ([]*engine.Attribute, error) {
+
+	resps, err := DoTxnRequest[GetTableColumnsResp](
+		ctx,
+		t.txnOperator,
+		true,
+		t.engine.anyShard,
+		OpGetPrimaryKeys,
```
Comment (reusee) [inline pkg/vm/engine/memoryengine/table.go:182]: OpGetTableColumns

**Review by reusee — CHANGES_REQUESTED** [review]

File: pkg/vm/engine/memoryengine/table.go (hunk 1 above)
Comment (iamlinjunhong) [reply pkg/vm/engine/memoryengine/table.go]: Done

**Review by fengttt — APPROVED** [review]

**Review by reusee — APPROVED** [review]

**Comment by fengttt** [comment]
@mergifyio refresh

**Pull request merged**